pallets/asset-manager/src/lib.rs

```diff
@@ -54,7 +54,7 @@ pub mod mock;
 pub mod tests;
 pub mod weights;
 
-#[pallet]
+#[pallet(dev_mode)]
```

Comment:
Why `dev_mode`? (Used in other pallets as well)

Context: https://github.com/paritytech/substrate/pull/12536

I'm not sure which feature(s) are relevant, but it overall seems inappropriate for production-grade pallets.

Reply:
Mostly did that in the beginning to reduce the noise from a gazillion warnings. We need to add call index to each of those pallet dispatchables. Will take care of it.